mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -191,6 +192,7 @@ CLASS none/sg net/minecraft/entity/Entity
 	METHOD aj isSwimming ()Z
 	METHOD ak updateSwimming ()Z
 	METHOD al onSwimmingStart ()V
+	METHOD ao isTouchingLava ()Z
```

Comment:
I'd prefer `isInLava` even though this is technically more correct. ( :+1: / :-1: ?)